specification/cognitiveservices/data-plane/Face/v1.0/Face.json

```diff
@@ -0,0 +1,2655 @@
+    "noseLeftAlarOutTip": {
+     "$ref": "#/definitions/Position"
+    },
+    "noseRightAlarOutTip": {
+     "$ref": "#/definitions/Position"
+    },
+    "upperLipTop": {
+     "$ref": "#/definitions/Position"
+    },
+    "upperLipBottom": {
+     "$ref": "#/definitions/Position"
+    },
+    "underLipTop": {
+     "$ref": "#/definitions/Position"
+    },
+    "underLipBottom": {
+     "$ref": "#/definitions/Position"
+    }
+   }
+  },
+  "FaceAttributes": {
+   "type": "object",
+   "description": "Face Attributes",
+   "properties": {
+    "age": {
+     "type": "number",
+     "description": "Age in years"
+    },
+    "gender": {
+     "type": "string",
+     "description": "Gender: male or female",
+     "enum": [
+      "male",
+      "female"
+     ]
+    },
+    "smile": {
+     "type": "number",
+     "description": "Smile intensity, a number between [0,1] ",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "glasses": {
+     "type": "string",
+     "description": "Glasses type. Possible values are 'noGlasses', 'readingGlasses', 'sunglasses', 'swimmingGoggles'. ",
+     "enum": [
+      "noGlasses",
+      "readingGlasses",
+      "sunglasses",
+      "swimmingGoggles"
+     ]
+    },
+    "facialHair": {
+     "$ref": "#/definitions/FacialHairProperties"
+    },
+    "headPose": {
+     "$ref": "#/definitions/HeadPoseProperties"
+    },
+    "emotion": {
+     "$ref": "#/definitions/EmotionProperties"
+    }
+   }
+  },
+  "FacialHairProperties": {
+   "type": "object",
+   "description": "",
+   "properties": {
+    "mustache": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "beard": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "sideburns": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    }
+   }
+  },
+  "HeadPoseProperties": {
+   "type": "object",
+   "description": "",
+   "properties": {
+    "roll": {
+     "type": "number"
+    },
+    "yaw": {
+     "type": "number"
+    },
+    "pitch": {
+     "type": "number"
+    }
+   }
+  },
+  "EmotionProperties": {
+   "type": "object",
+   "description": "",
+   "properties": {
+    "anger": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "contempt": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "disgust": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "fear": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "happiness": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "neutral": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "sadness": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    },
+    "surprise": {
+     "type": "number",
+     "minimum": 0,
+     "maximum": 1
+    }
+   }
+  },
+  "FindSimilarRequest": {
+   "type": "object",
+   "required": [
+    "faceId"
+   ],
+   "description": "",
```

Comment:
Even though you've specified this should be flattened, the description should never be empty. Some languages may not work the same.